content-tree.d.ts

```diff
@@ -211,17 +211,27 @@ export declare namespace ContentTree {
         credit: string;
         picture?: ImageSetPicture;
     }
+    type TableColumnSettings = {
+        hideOnMobile: boolean;
+        sortable: boolean;
+        sortType: "text" | "number" | "date" | "currency" | "percent";
+    };
     interface Table extends Parent {
         type: "table";
-        children: [Caption | TableHead | TableBody];
-    }
-    interface Caption {
-        type: "caption";
-    }
-    interface TableHead {
-        type: "table-head";
-    }
-    interface TableBody {
-        type: "table-body";
+        title: string;
+        footer: string;
+        stripes: boolean;
+        compact: boolean;
+        layoutWidth: string;
+        rows: number;
```

Comment:
**Suggestion:** I think we don't need rows here now. The logic to find the number of rows would be more complex than I'd like to have in the tagMappings, so we've put a helper function in cp-content-pipeline.

Reply:
yes, that's fair enough no need for both rows and columns because we have .length and columns